src/site/content/en/blog/maskable-icon/index.md

```diff
@@ -0,0 +1,97 @@
+description: |
+  A description of the article that will appear in search results.
+authors:
+  - tigeroakes
+date: 2019-12-13
+hero: hero.png
+alt: Icons contained inside white circles compared to icons covering its entire circle
+tags:
+  - post
+  - capabilities
+  - progressive-web-apps
+---
+
+## What are maskable icons? {: #what }
+
+If you've installed a Progressive Web App on a recent Android phone, you might notice the icon shows up with a white background. Android Oreo introduced adaptive icons, which display app icons in a variety of shapes across different device models. Icons that don't follow this new format are given white backgrounds.
+
+<figure class="w-figure">
+  <img src="homescreen-any.png" alt="" style="max-width: 400px;">
+  <figcaption class="w-figcaption">
+    PWA icons in white circles on Android
+  </figcaption>
+</figure>
+
+Maskable icons are a new icon format that give you more control and let your Progressive Web App use adaptive icons. If you supply a maskable icon, your icon can fill up the entire shape and look great on all Android devices. Firefox and Chrome have recently added support for this new format, and you can adopt it in your apps.
+
+<figure class="w-figure">
+  <img src="homescreen-maskable.png" alt="" style="max-width: 400px;">
+  <figcaption class="w-figcaption">
+    PWA icons covering the entire circle on Android
+  </figcaption>
+</figure>
+
+## Are my current icons ready?
+
+Since maskable icons need to support a variety of shapes, you supply an opaque image with some padding that the browser can later crop into the desired shape and size. It's best not to rely on any particular shape, since it can vary by browser and per platform.
+
+<figure class="w-figure w-figure--inline-right">
+  <video class="w-screenshot" autoplay loop muted playsinline aria-label="Different masks applied to a maskable icon">
+    <source src="fugu-mask.webm" type="video/webm; codecs=vp8">
+    <source src="fugu-mask.mp4" type="video/mp4; codecs=h264">
+  </video>
+  <figcaption class="w-figcaption">
+    Different platform specific shapes
+  </figcaption>
+</figure>
+
+Luckily, there's a well-defined and [standardized](https://w3c.github.io/manifest/#icon-masks) "minimum safe zone" that all platforms respect. The important parts of your icon, such as your logo, should be within a circular area in the center of the icon with a radius equal to 40% of the icon width. The outer 10% edge may be cropped.
+
+You can check which parts of your icons land within the safe zone with Chrome DevTools. With your Progressive Web App open, launch DevTools and navigate to the "Application" panel. In the "Icons" section, you can choose to "Show only the minimum safe area for maskable icons". Your icons will be trimmed so that only the safe area is visible. If your logo is visible within this safe area, you're good to go.
+
+<figure class="w-figure">
+  <img src="devtools.png" alt="" style="max-width: 500px;">
```

Comment:
```suggestion
  <img src="devtools.png" class="w-screenshot" alt="" style="max-width: 500px;">
```

Reply:
You may also want to increase the size of this image or remove the max-width entirely to let it be full width. I found the text a little hard to read.

It may also look nicer centered with the `w-figure--center` class.